internal/pkg/build/build.go

```diff
@@ -412,11 +430,15 @@ func (b *Build) Full(ctx context.Context) error {
 		}
 
 		// create stage file for /etc/resolv.conf and /etc/hosts
-		sessionResolv, err := createStageFile("/etc/resolv.conf", stage.b, "Name resolution could fail")
-		if err != nil {
-			return err
-		} else if sessionResolv != "" {
-			defer os.Remove(sessionResolv)
+		// skip, if there is an explicit --bind for resolv.conf
+		sessionResolv := ""
+		if !haveBindFor(stage.b.Opts.Binds, "/etc/resolv.conf") {
```

Comment:
While you're there, please do the same for /etc/hosts.conf just below.

Reply:
Done